preproccesing.py

```diff
@@ -0,0 +1,100 @@
+import json
+import multiprocessing
+import os
+from functools import partial
+from multiprocessing import Pool
+import argparse
+import cv2
+import numpy as np
+import open3d
+import configparser
+from imageio import imwrite
+
+
+def undistort(path, fx, fy, cx, cy, k1, k2, k3, k4, k5, k6, p1, p2):
+    image_ = cv2.imread(path, -1)
+    camera_matrix = np.array([[fx, 0, cx],
```

Comment:
You create this matrix per each image. Maybe create it outside the function and pass as parameter?